android/app/src/main/cpp/audio_engine.cpp

```
@@ -16,6 +17,10 @@
 // in the class body) can reference it without a forward declaration.
 static std::atomic<bool> g_muted{false};
 
+// Global JNI references for voice activity callbacks
+static JavaVM* g_jvm = nullptr;
```

Clang (14.0.6)

[warning] variable 'g_jvm' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)